myshopmatic.com.shopmatic.json

```diff
@@ -0,0 +1,20 @@
+{
+  "providerId": "myshopmatic.com",
+  "providerName": "myshopmatic.com",
+  "serviceId": "shopmatic",
+  "serviceName": "shopmatic",
+  "logoUrl": "",
+  "description": "Configures myshopmatic.com zone ",
+  "syncBlock": false,
+  "shared": false,
+  "warnPhishing": false,
+  "records": [
+    {
+      "groupId": "REDIRECT",
+      "type": "CNAME",
+      "host": "www",
+      "data": "%INPUTDOMAIN%"
+    }
+  ]
+}
+
```

Comment:
I'm really confused about shopmatic.  What is the company, host, and service?

Naming conventions:

Each company is a Service Provider.  

They get a providerId and a providerName.  The providerId should be a host (e.g. foo.com) and the name should be what they want shown in the UX (Foo Incorporated).

They need at least one service, but can have multiple.  These are idenfied by the serviceId/serviceName.

So for foo.com, they might have "mail" and "hosting".

The template for mail should be:

foo.com.mail.json with providerId = foo.com and serviceId = mail

For hosting

foo.com.hosting.json 

The shopmatic templates do not follow this convention and need to be fixed.